app.js

```diff
@@ -31,6 +31,10 @@ li.forEach((i) => {
   i.addEventListener('click', myToggle);
 });
 
+function saveData(data) {
+  const stringyData = JSON.stringify(data);
+  window.localStorage.setItem('customFormData', stringyData);
+}
 form.addEventListener('submit', (e) => {
```

Comment:
- [ ] Kindly change the event from `submit` on the form to `change` event on every input so every change the user made it the data will be updated and if he adds some data and left the page when he comes back he can continue what he left.